utils/slice.go

```diff
@@ -19,6 +23,56 @@ func IndexOf(strs []string, str string) int {
 	return -1
 }
 
+// NormalizeString normalizes a string as NFKC
+func NormalizeString(str string) string {
+	return norm.NFKC.String(str)
+}
+
+// ValidateString checks if the given string is:
+//
+// 1. non-empty
+// 2. entirely composed of utf8 runes
+// 3. normalized as NFKC
+// 4. does not contain any forbidden Unicode code points
+func ValidateString(str string, forbidden string) error {
+	return validateString(str, false, forbidden)
+}
```

Comment:
```suggestion
func ValidateString(str string, forbidden ...string) error {
    if len(forbidden) == 0 {
        forbidden = utils.DefaultDelimiter
    } else {
        forbidden = strings.Join(forbidden)
    }

    return validateString(str, false, forbidden)
}
```
If we do the above, we can avoid having to pass `_` to all calls of `ValidateString`. And since we want to enforce this delimiter going forward, it'll be less error prone.